CONTRIBUTING.md

```diff
@@ -34,7 +34,6 @@ Thank you for considering contributing to **dlt**! We appreciate your help in ma
 1. Fork the `dlt` repository and clone it to your local machine.
 2. Install `uv` with `make install-uv` (or follow the [official instructions](https://docs.astral.sh/uv/getting-started/installation/)).
 3. Run `make dev` to install all dependencies, including development ones.
-4. Activate your virtual environment with `make shell` and start working, or prepend all commands with `uv run` to run them within the uv environment. `uv run` is encouraged, it automatically keeps your project dependencies up to date.
```

Comment:
The stuff about uv is still acurate and should not be deleted. I would probably keep this info as is and re-add a shell command which activates the virtual env: `source .venv/bin/activate`

Reply:
Ah yes, ok :)